gui/wxpython/gcp/manager.py

```diff
@@ -891,8 +891,8 @@ def OnSrcSelection(self, event):
             if p.returncode == 0:
                 print("returncode = ", str(p.returncode))
                 self.parent.Map.region = self.parent.Map.GetRegion()
-        except:
-            pass
+        except Exception as e:
```

Comment:
Looking more into this code, this has probably never run properly, see the usage of `src_map`  as a string instead of the variable. So I suggest to remove the entire try except block and its content.

Reply:
Got it, I will remove the entire `try-except` block